extensions/dicom-video/package.json

```diff
@@ -21,7 +21,6 @@
   },
```

Comment:
Are we removing the dicom-video extension? I'm all for moving it out, but has this been discussed?

Reply:
I haven't remove it, it was using trackingIdentifier context which it shouldn't, I removed that part.